### ISSUE TITLE
SMAR-560 ⁃ Analyse current ssl certificate and generate new CSR

### ISSUE THREAD

**Issue by thatdoogieguy** [thatdoogieguy/SST#443]
Analyse current ssl certificate and generate new CSR

┆Attachments: <a href="https://smartsensor.atlassian.net/secure/attachment/25041/dashboard.smartsensor.png">dashboard.smartsensor.png</a> | <a href="https://smartsensor.atlassian.net/secure/attachment/25043/gcbbq.smartsensor.png">gcbbq.smartsensor.png</a> | <a href="https://smartsensor.atlassian.net/secure/attachment/25042/services.smartsensor.png">services.smartsensor.png</a>

**Comment by thatdoogieguy** [comment]
➤ Michael Effendi commented:

I Need to know where domain 'MAPMYBINS.COM.AU' is hosted before creating csr file. Waiting reply from colby about server details.

**Comment by thatdoogieguy** [comment]
➤ Michael Effendi commented:

CSR certificate has been generated for new ssl certificate. Waiting new certificate to be implemented on old server.

**Comment by thatdoogieguy** [comment]
➤ Michael Effendi commented:

SSL certificate has been implemented into smartsensor domain web browser. Please refer to the attachments for details.

I'll create new ticket for mapmybins domain once colby is ready to provide me credentials to login to the web browser.
